### PR TITLE
288 guard against over voting update

### LINKED ISSUE #288: Calculations Based Triggers: Guard against over voting update
### Current requirement functionality
---
*This functionality exists in the current software.*
### User Story
---
The number of votes exceeds the number of voters on the station voter list: the total number of valid, invalid, unstamped and discarded/canceled ballots is more than the total number of voters/registrants on the polling station list.

### Implementation Plan
---
Update all quarantine checks with a new `status` field that can be used to enable or disable the check/trigger.

### PR DESCRIPTION
**Pull request by JohnMwashuma**
Fixes #288 

- [x] Add/Update tests

### CODE REVIEW

File: tally_ho/apps/tally/templates/super_admin/quarantine_checks_list.html [hunk 1]
```diff
@@ -20,6 +20,7 @@ <h1>{% trans 'Quarantine checks List' %}</h1>
         <th>{% trans 'Modification date' %}</th>
         <th>{% trans 'Value' %}</th>
         <th>{% trans 'Percentage' %}</th>
+        <th>{% trans 'Status' %}</th>
         <th>{% trans 'Actions' %}</th>
     </tr>
 {% for oneCheck in checks %}
```
Comment (pld) [inline tally_ho/apps/tally/templates/super_admin/quarantine_checks_list.html:26]: we can simply call this `check in checks`

File: tally_ho/libs/verify/quarantine_checks.py [hunk 2]
```diff
@@ -17,8 +17,19 @@ def create_quarantine_checks():
...
 def quarantine_checks():
     """Return tuples of (QuarantineCheck, validation_function)."""
-    methods = [pass_overvote, pass_tampering]
-    checks = QuarantineCheck.objects.all().order_by('pk')
+    all_methods = {'pass_overvote': pass_overvote,
+                   'pass_tampering': pass_tampering}
+    methods = []
+
+    quarantine_checks_methods =\
+        QuarantineCheck.objects.filter(
```
Comment (pld) [inline tally_ho/libs/verify/quarantine_checks.py:25]: do we want to order this?

Comment (pld) [reply tally_ho/libs/verify/quarantine_checks.py:25]: perhaps alphabetically

Comment (JohnMwashuma) [reply tally_ho/libs/verify/quarantine_checks.py:25]: Nice catch, forgot that and i will order by the primary key as we were doing initially when querying all checks. https://github.com/onaio/tally-ho/blob/cc0bff7b63825e828425327a28f872af3e3fa4f1/tally_ho/libs/verify/quarantine_checks.py#L21

File: tally_ho/apps/tally/templates/super_admin/quarantine_checks_list.html (hunk 1 above)
Comment (JohnMwashuma) [reply tally_ho/apps/tally/templates/super_admin/quarantine_checks_list.html]: Done

File: tally_ho/libs/verify/quarantine_checks.py (hunk 2 above)
Comment (JohnMwashuma) [reply tally_ho/libs/verify/quarantine_checks.py:25]: Done

**Pull request merged**